tests/test_custom_objects_rest.py

```diff
@@ -0,0 +1,100 @@
+"""
+Test that all fields can be replicated for a stream that is a custom object (REST API)
+"""
+from datetime import datetime, timedelta
+
+from tap_tester import connections, menagerie, runner
+
+from base import SalesforceBaseTest
+
+
+class SalesforceCustomObjects(SalesforceBaseTest):
+    """Test that all fields can be replicated for a stream that is a custom object"""
+
+    # Note: custom record as seen Aug 9, 2023
+    # TODO do we need to hard code start date to 08-04?
+    # 'CreatedDate': '2023-08-04T20:13:50.000000Z'
+    # 'SystemModstamp': '2023-08-04T20:13:50.000000Z',
+    # 'LastModifiedDate': '2023-08-04T20:13:50.000000Z',
+    # 'LastReferencedDate': '2023-08-09T20:19:45.000000Z',
+    # 'LastViewedDate': '2023-08-09T20:19:45.000000Z',
+    start_date =  (datetime.now() + timedelta(days=-1)).strftime("%Y-%m-%dT00:00:00Z")
```

Comment:
Your TODO and the dates of these records look like yesterday won't work for long.  Is this a static record where we need a hardcoded start date or are we okay?

Reply:
I don't know.  The reason all these dates are still commented there was the initial effort to understand why the record is still being replicated since the start date is now well past the static record creation time.  I was going to change the start date but since it wasn't technically broken I didn't fix it.

Reply:
Follow up card to be written to determine what the expected behavior is for custom objects with no defined replication method.  Determine how to set replication method and update test to expect correct behavior.

Reply:
https://jira.talendforge.org/browse/TDL-23886